core/storage/base_model/gae_models.py

```diff
@@ -346,6 +434,19 @@ def commit(self, committer_id, commit_message, commit_cmds):
 
     @classmethod
     def revert(cls, model, committer_id, commit_message, version_number):
+        """Reverts model to previous version.
+
+        Args:
+            model: VersionedModel.
+            committer_id: str.
+            commit_message: str.
+            version_number: str. Version to revert to.
```

Comment:
I noticed that this appears to sometimes be a str, and sometimes be an int. You might be right, but I just wanted to double check if you've tested this carefully and are confident about the type information that you're including here?